packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## 45.5.0 - March 7, 2019
+
+### Added
+
+#### salesforcedx-vscode-apex
+
+- Show Apex Javadoc information when hovering on symbols ([PR #1106](https://github.com/forcedotcom/salesforcedx-vscode/pull/1106))
```

Comment:
i would change "Apex Javadoc" to "Apex doc comment"

Reply:
But doesn't the comment have to follow the Javadoc format ?

Reply:
I don't think we officially say Javadoc anywhere in our public documentation. Also, unlike Java doc comments or C# doc comments we don't explicitly support any tags. I think Javadoc is just a format that some customers are using to document Apex source. As for implementation we right now just show any block comments associated with a symbol definition we don't validate or process any of the tags that are used.

Reply:
Ok, let me change it to something like Apex document block comment to reflect what we currently support.

Reply:
Does it use the [ApexDoc format](https://github.com/SalesforceFoundation/ApexDoc#documenting-class-files)? Should we say "ApexDoc"?

Reply:
Well, what Prakash mentioned about no support for tags, etc would still apply. Also, that ApexDoc format is not official, right ? By official I mean part of the Apex language.

Reply:
Yeah, ApexDoc is a community-driven thing. The "Show Apex block comment information" version you added works, @lcampos .